core/src/test/scala/org/apache/spark/storage/FallbackStorageSuite.scala

```diff
@@ -107,6 +106,51 @@ class FallbackStorageSuite extends SparkFunSuite with LocalSparkContext {
     FallbackStorage.read(conf, ShuffleBlockId(1, 2L, 0))
   }
 
+  test("fallback storage APIs - readFully") {
```

Comment:
SPARK-39200: fallback storage APIs - readFully

Reply:
+1 for @yaooqinn 's comment.